### PR TITLE
i18n(es): Update `guides/` `V3`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Closes # 
- What does this PR change? Give us a brief description.
Update `guides/` `V3` [PR](https://github.com/withastro/docs/pull/4166/files#diff-f3d2bb9ef8c11d189abbf550bf423e5698c35a1af013173d764613cdbb15aaa9)

### CODE REVIEW

File: src/content/docs/es/guides/rss.mdx [hunk 1]
```diff
@@ -76,11 +76,11 @@ Puedes generar esta matriz a partir de un esquema de colección de contenido o u
...
 
 ```js title="src/pages/rss.xml.js" "items:" "const blog = await getCollection('blog');"
-export async function get(context) {
+export async function GET(context) {
   const blog = await getCollection('blog');
   return rss({
-    title: 'Buzz’s Blog',
-    description: 'A humble Astronaut’s guide to the stars',
+    title: 'Blog de Buzz',
+    description: 'Guía de las estrellas para un humilde astronauta',
```
Comment (dreyfus92) [inline src/content/docs/es/guides/rss.mdx:83]:
```suggestion
    description: 'Guía de un humilde astronauta a las estrellas',
```

File: src/content/docs/es/guides/rss.mdx [hunk 2]
```diff
@@ -47,20 +47,20 @@ El archivo de ejemplo a continuación `src/pages/rss.xml.js` creará un feed RSS
...
 
-export function get(context) {
+export function GET(context) {
   return rss({
     // `<title>` campo en el xml generado
-    title: 'Buzz’s Blog',
+    title: 'Blog de Buzz',
     // `<description>` campo en el xml generado
-    description: 'A humble Astronaut’s guide to the stars',
+    description: 'Guía de las estrellas para un humilde astronauta',
```
Comment (dreyfus92) [inline src/content/docs/es/guides/rss.mdx:55]:
```suggestion
    description: 'Guía de un humilde astronauta a las estrellas',
```

File: src/content/docs/es/guides/rss.mdx [hunk 3]
```diff
@@ -120,10 +120,10 @@ Esta función asume, pero no verifica, que todas las propiedades RSS necesarias
 
 ```js title="src/pages/rss.xml.js" "pagesGlobToRssItems" "await pagesGlobToRssItems("
 import rss, { pagesGlobToRssItems } from '@astrojs/rss';
-export async function get(context) {
+export async function GET(context) {
   return rss({
-    title: 'Buzz’s Blog',
-    description: 'A humble Astronaut’s guide to the stars',
+    title: 'Blog de Buzz',
+    description: 'Guía de las estrellas para un humilde astronauta',
```
Comment (dreyfus92) [inline src/content/docs/es/guides/rss.mdx:126]:
```suggestion
    description: 'Guía de un humilde astronauta a las estrellas',
```

File: src/content/docs/es/guides/rss.mdx [hunk 4]
```diff
@@ -177,12 +178,12 @@ Cuando utilices imports glob con Markdown, puedes utilizar el helper `compiledCo
...
 import sanitizeHtml from 'sanitize-html';
-export function get(context) {
+export function GET(context) {
   const postImportResult = import.meta.glob('../posts/**/*.md', { eager: true }); 
   const posts = Object.values(postImportResult);
   return rss({
-    title: 'Buzz’s Blog',
-    description: 'A humble Astronaut’s guide to the stars',
+    title: 'Blog de Buzz',
+    description: 'Guía de las estrellas para un humilde astronauta',
```
Comment (dreyfus92) [inline src/content/docs/es/guides/rss.mdx:186]:
```suggestion
    description: 'Guía de un humilde astronauta a las estrellas',
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Te dejo algunas sugerencias @Waxer59, buen trabajo 😁

**Comment by Waxer59** [comment]
Listo!, gracias por la review 🤠

**Comment by Waxer59** [comment]
Commit: 
* [`355b66`](https://github.com/withastro/docs/pull/4166/commits/355b667be370620c87c204a2608a61bc55bd384e)

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**